tests/scalers/azure-pipelines.test.ts

```diff
@@ -69,13 +69,16 @@ test.serial('PoolID: Deployment should scale to 1 replica after queueing job', a
   var definitionID = parseInt(buildDefinitionID)
 
   await build.queueBuild(null, projectName, null, null, null, definitionID)
+  await sleep(60000);
```

Comment:
Do you plan to do the migration to Go in this PR?

Reply:
I didn't have, but I'll do it. This and application insights e2e tests